README.md

```diff
@@ -39,7 +39,7 @@ const menuMiddleware = new MenuMiddleware('/', menuTemplate);
 bot.command('start', (ctx) => menuMiddleware.replyToContext(ctx));
 bot.use(menuMiddleware);
 
-bot.launch();
+bot.start();
```

Comment:
Shouldn't it also be awaited?

```suggestion
await bot.start();
```

Reply:
> Shouldn't it also be awaited?

no need to await .
see: https://t.me/grammyjs/188199

Reply:
While it’s not necessary it’s still bad code in my opinion. Promises should always be awaited. Otherwise they might result in unhandled promise rejection warnings. This was a pitfall especially on older Node.js versions.

For grammY in this specific case the start would only fail on the initial connection so it’s not as bad but I still prefer correct code over „would also work without“.